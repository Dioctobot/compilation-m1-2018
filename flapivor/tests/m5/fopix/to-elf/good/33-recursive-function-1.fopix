def fib (x) =
  if x = 0 || x = 1 then 1 else fib (x - 1) + fib (x - 2) end

val main =
    print_int (fib (10))