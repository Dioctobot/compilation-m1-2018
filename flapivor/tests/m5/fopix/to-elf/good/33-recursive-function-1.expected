Process exited with status exited(0).
STDOUT:
89
STDERR:



